java/src/main/java/com/microsoft/graph/bulk/impl/BulkGremlinObjectMapper.java

```diff
@@ -0,0 +1,29 @@
+package com.microsoft.graph.bulk.impl;
```

Comment:
We don't include `microsoft` in the package structure anymore, because of the rebranding of azure. 
Please change the package structure to `com.azure. ...`